public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_configurations_panel.scss

```diff
@@ -39,4 +39,50 @@
   &.showSecondary > .wizConfig__section {
     transform: translateX(-100%);
   }
+}
```

Comment:
have we verified and changes on these will not affect core visualize?

Reply:
To the best of my knowledge, it shouldn't. The only code in observability that has this classname was the code I've added